android-maps-utils-ktx/src/test/java/com/google/maps/ktx/heatmap/HeatmapTest.kt

```diff
@@ -0,0 +1,16 @@
+package com.google.maps.ktx.heatmap
+
+import com.google.android.gms.maps.model.LatLng
+import com.google.maps.android.heatmaps.WeightedLatLng
+import com.google.maps.ktx.heatmaps.toWeightedLatLng
+import org.junit.Assert.*
+import org.junit.Test
+
+class HeatmapTest {
+    @Test
+    fun `to WeightedLatLng converts correctly`() {
+        val latLng = LatLng(1.0, 2.0)
+        assertEquals(WeightedLatLng(latLng), latLng.toWeightedLatLng())
```

Comment:
Looks like Travis builds are failing due to this test failure:

~~~
> Task :android-maps-utils-ktx:testDebugUnitTest
com.google.maps.ktx.heatmap.HeatmapTest > to WeightedLatLng converts correctly FAILED
    java.lang.AssertionError at HeatmapTest.kt:13
~~~